onnxruntime/python/tools/transformers/models/llama/README.md

```diff
@@ -79,8 +81,15 @@ model.save_pretrained(name.split("/")[-1] + "-onnx")
 
 Here are some additional examples for exporting LLaMA-2.
 
-Export Saved Model on Disk
+Export Model with Different GPU Device Ids
 ```
+# From source using first GPU:
+$ CUDA_VISIBLE_DEVICES=0 python3 -m models.llama.convert_to_onnx -m meta-llama/Llama-2-7b-hf --input ./Llama-2-7b-hf --output ./llama2-7b
+
+# From wheel using second GPU:
+$ CUDA_VISIBLE_DEVICES=1 python3 -m onnxruntime.transformers.models.llama.convert_to_onnx -m meta-llama/Llama-2-7b-hf --input ./Llama-2-7b-hf --output ./llama2-7b
+Export Saved Model on Disk
```

Comment:
The "Export Model with Different GPU Device Ids" and "Export Saved Model on Disk" sections appear to be merged when previewing the README. I think you can close the code block after the `CUDA_VISIBLE_DEVICES=1` line and then start the new code block after the `Export Saved Model on Disk` line.